cupy/logic/comparison.py

```diff
@@ -1,4 +1,6 @@
 from cupy import core
+from cupy.creation.from_data import asanyarray
+from numpy import complex64, complex128
```

Comment:
@toslunar no problem, so how about changing these two lines to
```python
import cupy.creation.from_data 
import numpy
```
and refer to the function and constants using the full identifiers? Thanks!

Reply:
`cupy.asanyarray` can be used in functions because `import cupy` finishes first.

Then group(*1) and sort(*2) imports:
```
import numpy

import cupy
from cupy import core
```

*1:
> 1. Standard library imports.
> 2. Related third party imports.
> 3.  Local application/library specific imports.

https://www.python.org/dev/peps/pep-0008/#imports

*2:
> [H306] Alphabetically order your imports by the full module path. [...]

https://docs.openstack.org/hacking/latest/user/hacking.html#imports

Reply:
Thank you, pushing a commit shortly.

Reply:
done